client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -114,12 +114,27 @@
               editor-id="recommendationText"
               :procedure-id="procedureId"
               @insert="text => modalProps.handleInsertText(text)" />
-            <dp-recommendation-modal
-              v-if="segment.hasRelationship('tags')"
+            <dp-modal
               ref="recommendationModal"
-              :procedure-id="procedureId"
-              :segment-id="segment.id"
-              @insert-recommendation="text => modalProps.appendText(text)" />
+              class="recommendation-modal"
+              content-classes="u-2-of-3">
+              <div class="display--inline-block">
+                <h3 class="u-mb">
+                  {{ Translator.trans('segment.recommendation.insert.similar') }}
+                  <dp-icon icon="ai" />
```

Comment:
well... i can't guess how the UI looks, straight from the code... but the AI icon should be reserved for places that are really ai-powered which is only the case for the oracle.recommendation tab. i suggest we look into that tomorrow.

Reply:
the ai icon has been removed here https://github.com/demos-europe/demosplan-core/pull/1486/commits/564be88fd18b61f2f5f15b60268a73569a079c64 and will be added in the corresponding addon directly